README.md

```diff
@@ -31,18 +31,24 @@ For documentation about the Jellyfin app, see [here](https://github.com/jellyfin
 
    - (Optional) You can provide preferred [jellyfin-tizen-builds](https://github.com/jeppevinkel/jellyfin-tizen-builds) option (Jellyfin / Jellyfin-TrueHD / Jellyfin-master / Jellyfin-master-TrueHD / Jellyfin-secondary) as second argument. By default, Jellyfin option is used.
    - (Optional) You can provide preferred [jellyfin-tizen-builds releases](https://github.com/jeppevinkel/jellyfin-tizen-builds/releases) release tag URL as third argument. By default, latest version is used. This is useful if you want to install older Jellyfin Tizen Client version.
+   - (Optional) You can provide a custom Samsung certificate by mounting the `.p12` files at `/certificates/` and providing the certificate password as fourth argument.
    - If you do not want to use either of these options and just install the default build, do not put anything after the IP address.
 
 ```bash
-docker run --rm ghcr.io/georift/install-jellyfin-tizen <samsung tv ip> <build option> <tag url>
+docker run --rm ghcr.io/georift/install-jellyfin-tizen <samsung tv ip> <build option> <tag url> <certificate password>
 ```
 
-Example:
+Examples:
 
 ```bash
 docker run --rm ghcr.io/georift/install-jellyfin-tizen 192.168.0.10 Jellyfin-TrueHD "https://github.com/jeppevinkel/jellyfin-tizen-builds/releases/tag/2024-05-13-0139"
 ```
 
+
+```bash
+docker run --rm -v "$(pwd)/author.p12" -v "$(pwd)/distributor.p12" ghcr.io/georift/install-jellyfin-tizen 192.168.0.10 Jellyfin "" 'CertPassw0rd!' # Third argument empty to use latest tag
```

Comment:
I suggest mounting the certificates to the container `/certificates` directory since the source could be anywhere, e.g.:
```
-v "$(pwd)/author.p12":/certificates/author.p12 -v "$(pwd)/distributor.p12":/certificates/distributor.p12
```

Reply:
Oh yes, you are right I will update this line